docs/samples/ble/raspberry-esp32-xiaomi-sensor-htu21d.md

```diff
@@ -0,0 +1,248 @@
+I (150) esp_image: segment 2: paddr=0x00036010 vaddr=0x40080000 size=0x00400 (  1024) load
+0x40080000: _WindowOverflow4 at /home/yulia/esp/esp-idf/components/freertos/xtensa_vectors.S:1779
+
+I (152) esp_image: segment 3: paddr=0x00036418 vaddr=0x40080400 size=0x09bf8 ( 39928) load
+I (177) esp_image: segment 4: paddr=0x00040018 vaddr=0x400d0018 size=0x71ea0 (466592) map
+0x400d0018: _flash_cache_start at ??:?
+
+I (341) esp_image: segment 5: paddr=0x000b1ec0 vaddr=0x40089ff8 size=0x08d50 ( 36176) load
+0x40089ff8: r_lld_pdu_rx_handler at ??:?
+
+I (367) boot: Loaded app from partition at offset 0x10000
+I (367) boot: Disabling RNG early entropy source...
+I (368) cpu_start: Pro cpu up.
+I (371) cpu_start: Application information:
+I (376) cpu_start: Project name:     gatt_server_demos
+I (382) cpu_start: App version:      b2732e4-dirty
+I (387) cpu_start: Compile time:     Jun  3 2019 17:06:46
+I (393) cpu_start: ELF file SHA256:  2d3d593eed804970...
+I (399) cpu_start: ESP-IDF:          v4.0-dev-584-g3bf56cdd1
+I (406) cpu_start: Starting app cpu, entry point is 0x400810e0
+0x400810e0: call_start_cpu1 at /home/yulia/esp/esp-idf/components/esp32/cpu_start.c:267
+
+I (0) cpu_start: App cpu up.
+I (416) heap_init: Initializing. RAM available for dynamic allocation:
+I (423) heap_init: At 3FFAFF10 len 000000F0 (0 KiB): DRAM
+I (429) heap_init: At 3FFB6388 len 00001C78 (7 KiB): DRAM
+I (435) heap_init: At 3FFB9A20 len 00004108 (16 KiB): DRAM
+I (441) heap_init: At 3FFBDB5C len 00000004 (0 KiB): DRAM
+I (447) heap_init: At 3FFC8EA0 len 00017160 (92 KiB): DRAM
+I (453) heap_init: At 3FFE0440 len 00003AE0 (14 KiB): D/IRAM
+I (460) heap_init: At 3FFE4350 len 0001BCB0 (111 KiB): D/IRAM
+I (466) heap_init: At 40092D48 len 0000D2B8 (52 KiB): IRAM
+I (472) cpu_start: Pro cpu start user code
+I (155) cpu_start: Starting scheduler on PRO CPU.
+I (0) cpu_start: Starting scheduler on APP CPU.
+I (157) GATTS_DEMO: starting ESP gatt demo
+
+```
+
+## Preparing MI Sensor
+Install the battery in sensor.
+
+## Installing gateway scripts on Raspberry Pi
+Login to your Raspberry PI using SSH and install demo. It requires at least **python3.4**. 
+```
+sudo apt-get install libglib2.0-dev
+pip3 install tb-ble-adapter
+```
+Run `pip3 show tb-ble-adapter` to find the location of installed demo:
+```
+$> pip3 show tb-ble-adapter
+...
+Location: /home/yulia/.local/lib/python3.6/site-packages
+...
+```
+
+## Running main program
+Run `sudo python3 tb-ble-adapter.py` with required arguments and enjoy (ﾉ◕ヮ◕)ﾉ*:･ﾟ
+
+## Troubleshooting
+In order to perform troubleshooting, you must check demo script output. It displays connection status with every device, as well as received and sent data. 
+
+## Device provisioning
+Run python script **before** importing dashboard. For this you have to create gateway device and specify it in command arguments:
+```
+sudo python3 tb-ble-adapter.py -t *your_gateway_access_token*
+```
```

Comment:
Add here example logs to demonstrate device scanning, discovery of known devices and device connection